src/plugins/saved_search/public/services/saved_searches/to_saved_search.test.ts

```diff
@@ -21,12 +20,7 @@ const mockServices = {
   spaces: spacesPluginMock.createStartContract(),
   embeddable: {
     getAttributeService: jest.fn(
-      (_, opts) =>
-        new AttributeService(
-          SEARCH_EMBEDDABLE_TYPE,
-          coreMock.createStart().notifications.toasts,
-          opts
-        )
+      (_, opts) => new AttributeService('search', coreMock.createStart().notifications.toasts, opts)
```

Comment:
Is this change related to the app menu?

Reply:
At some point of the implementation it was causing circular deps. It's not anymore but I decided to keep the change since it's only the test which was using this dep. Should I revert?